rs-e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/rse2e/external/hapi/HapiHL7FileMatcherTest.groovy

```diff
@@ -35,8 +35,8 @@ class HapiHL7FileMatcherTest extends Specification {
         def mockInputMessage2 = Mock(Message)
         def mockOutputMessage1 = Mock(Message)
         def mockOutputMessage2 = Mock(Message)
-        spyFileMatcher.mapMessageByControlId(mockInputFiles) >> [ "001": mockInputMessage1, "002": mockInputMessage2 ]
-        spyFileMatcher.mapMessageByControlId(mockOutputFiles) >> [ "001": mockOutputMessage1, "002": mockOutputMessage2 ]
+        spyFileMatcher.parseAndMapMessageByControlId(mockInputFiles) >> ["001": mockInputMessage1, "002": mockInputMessage2 ]
```

Comment:
Isn't `parseAndMapMessageByControlId` now returning a map of string to `HapiHL7Message` now?  This test here, and below, are still returning a map of a string to `Message`.  Shouldn't this be updated?

Reply:
Done